src/App.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import './App.css'
+
+function App() {
+  const [times, setTimes] = useState(0)
+
+  const handleClick = () =>{
+    setTimes(prevTime => prevTime + 1)
+  }
```

Comment:
Since the logic in this function fits on one line, it would be better to remove the brackets to use less lines of code. like this,
```suggestion
  const handleClick = () =>setTimes(prevTime => prevTime + 1)
```